source/aggregation-tutorials.txt

```diff
@@ -0,0 +1,112 @@
+.. _pymongo-aggregation-tutorials-landing:
+
+=====================
+Aggregation Tutorials
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: python, code example, runnable app
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. toctree::
+
+   /aggregation-tutorials/filtered-subset/
+   /aggregation-tutorials/group-total/
+   /aggregation-tutorials/unpack-arrays/
+   /aggregation-tutorials/one-to-one-join/
+   /aggregation-tutorials/multi-field-join/
+
+Overview
+--------
+
+Aggregation tutorials provide detailed explanations of common
+aggregation tasks in a step-by-step format. The tutorials are adapted
+from examples in the `Practical MongoDB Aggregations book
+<https://www.practical-mongodb-aggregations.com/>`__ by Paul Done.
+
+Each tutorial includes the following sections:
+
+- **Introduction**, which describes the purpose and common use cases of the
+  aggregation type. This section also describes the example and desired
+  outcome that the tutorial demonstrates.
+
+- **Before You Get Started**, which describes the necessary databases,
+  collections, and sample data that you must have before building the
+  aggregation pipeline and performing the aggregation.
+
+- **Tutorial**, which describes how to build and run the aggregation
+  pipeline. This section describes each stage of the completed
+  aggregation tutorial, and then explains how to run and interpret the
+  output of the aggregation.
+
+At the end of each aggregation tutorial, you can find a link to a fully
+runnable Python code file that you can run in your environment.
+
+.. _pymongo-agg-tutorial-template-app:
+
+Aggregation Template App
+------------------------
+
+Before you begin following an aggregation tutorial, you must set up a
+new Python app. You can use this app to connect to a MongoDB
+deployment, insert sample data into MongoDB, and run the aggregation
+pipeline in each tutorial.
+
+.. tip:: 
+   
+   To learn how to install the driver and connect to MongoDB,
+   see :ref:`pymongo-get-started`
+
+Once you install the driver, create a file called
+``agg_tutorial.py``. Paste the following code in this file to create an
+app template for the aggregation tutorials:
+
+.. literalinclude:: /includes/aggregation/template-app.js
+   :language: python
+   :copyable: true
```

Comment:
All examples in this PR will be JS code until we translate them to Python just before release